M2/Macaulay2/packages/HomologicalShiftIdeals.m2

```diff
@@ -0,0 +1,1001 @@
+-- -*- coding: utf-8 -*-
+newPackage(
+        "HomologicalShiftIdeals",
+        Version => "1.0", 
+        Date => "May 9, 2022",
+        Authors => {{Name => "Antonino Ficarra", Email => "antficarra@unime.it"}},
+        Headline => "A Macaulay2 package for computing the homological shift ideals of a monomial ideal of a polynomial ring",
```

Comment:
Package headlines must have fewer than 100 characters.  I'd probably drop "A Macaulay2 package for computing the" from the beginning.